src/SonarScanner.MSBuild.TFS/CoverageReportProcessorBase.cs

```diff
@@ -95,16 +109,20 @@ private bool TryConvertCoverageReports(IEnumerable<string> vscoverageFilePaths,
         {
             var xmlFileNames = vscoverageFilePaths.Select(x => Path.ChangeExtension(x, XmlReportFileExtension));
 
-            Debug.Assert(!xmlFileNames.Any(x => File.Exists(x)),
-                "Not expecting a file with the name of the binary-to-XML conversion output to already exist.");
-
-            var anyFailedConversion = vscoverageFilePaths.Zip(xmlFileNames, (x, y) => new { coverage = x, xml = y })
-                .Any(x => !this.converter.ConvertToXml(x.coverage, x.xml));
-
-            if (anyFailedConversion)
+            if (xmlFileNames.Any(x => File.Exists(x)))
+            {
+                Logger.LogWarning("Not expecting a file with the name of the binary-to-XML conversion output to already exist.");
+            }
+            else
             {
-                vscoveragexmlPaths = Enumerable.Empty<string>();
-                return false;
+                var anyFailedConversion = vscoverageFilePaths.Zip(xmlFileNames, (x, y) => new { coverage = x, xml = y })
```

Comment:
FYI personally, I dislike this style of functional programming; it's overly complex, harder to read and harder to debug.
If you do need make changes to this part of the function, feel free to unwrap it into a simple `for-each`.

Reply:
Will be modified for #735